dashboard/app/models/concerns/user/age.rb

```diff
@@ -0,0 +1,54 @@
+module User::Age
+  extend ActiveSupport::Concern
+
+  AGE_DROPDOWN_OPTIONS = (4..20).to_a << "21+"
+
+  included do
+    defer_age = proc {|user| %w(google_oauth2 clever).include?(user.provider) || Services::User::Migrated.new(user).call || Policies::Lti.lti?(user)}
```

Comment:
Call the new `Services::User::Migrated` service here, so we're not calling the `migrated?` method defined in user.rb.